apps/cow-amm-deployer/src/components/Button.tsx

```diff
@@ -73,6 +73,25 @@ export const PREDEFINED_CLASSES = {
       dark: "hover:bg-accent hover:text-accent-foreground",
     },
   },
+  tomato: {
```

Comment:
🤔 should we try to use the component from @bleu-ui here? wdyt @luizakp @devjoaov ?